web/js/codeworld.js

```diff
@@ -585,12 +583,13 @@ function setMode(force) {
 }
 
 function getCurrentProject() {
-    const doc = window.codeworldEditor.getDoc();
-    return {
-        'name': window.openProjectName || 'Untitled',
-        'source': doc.getValue(),
-        'history': doc.getHistory()
-    };
+  const doc = window.codeworldEditor.getDoc()
```

Comment:
Let's stick with semicolons after lines.  I believe running format.sh will add them for you.